### PR TITLE
feat(additionalQuestion): support additional questions for commit

### PR DESCRIPTION
**Pull request by parveen14**
You can add n number of additional questions for commit.
And append  answers to the body of commit message.
These question can help you to create smart-commit and pass information to third party like jira (with help of mapping)

### CODE REVIEW

**Review by samarpan-b — APPROVED** [review]

**Comment by samarpan-b** [comment]
@leonardoanalista Can you please check this PR and see if ti can be merged soon ? We are in need of this feature urgently.

File: .cz-config.js [hunk 1]
```diff
@@ -25,7 +25,20 @@ module.exports = {
     { value: 'revert', name: 'revert:   Revert to a commit' },
     { value: 'WIP', name: 'WIP:      Work in progress' },
   ],
-
+  additionalQuestions: [ 
+    {
```
[the remark this answers is not in the document]
Comment (leonardoanalista) [reply .cz-config.js:29]: Yes, I agree. You could add it to the README docs for this field and leave the other files as is because most people won't use it straight away.

I believe the subject, body, and footer are ideal for most cases and are already decent/solid defaults. 

The new field `additionalQuestions` could exist, no problems with it.

We need to thing about the initial API and keep it as simple as possible for someone new starting to use this package. We don't want them to feel overwhelmed by the number of options.

[the remark this answers is not in the document]
Comment (parveen14) [reply .cz-config.js]: @samarpan-b @leonardoanalista  Yes, got it.
changes done and `additionalQuestions`  removed from .cz-config.js  as its optional field.
we have `additionalQuestions`  in readme and config example file.

File: .cz-config.js [hunk 2]
```diff
@@ -25,13 +25,12 @@ module.exports = {
...
     { value: 'WIP', name: 'WIP:      Work in progress' },
   ],
-
   scopes: [{ name: 'accounts' }, { name: 'admin' }, { name: 'exampleScope' }, { name: 'changeMe' }],
 
   allowTicketNumber: false,
   isTicketNumberRequired: false,
   ticketNumberPrefix: 'TICKET-',
-  ticketNumberSuffix:'',
+  ticketNumberSuffix: '',
```
Comment (samarpan-b) [inline .cz-config.js:33]: Can you please revert these changes ?

File: README.md [hunk 3]
```diff
@@ -201,4 +220,4 @@ my items are:
 
 
 
-Leonardo Correa
+Leonardo Correa
```
Comment (samarpan-b) [inline README.md:223]: why this ?

Comment (parveen14) [reply README.md:221]: no changes done on this line , not sure why it showing as same line removed & added

File: cz-config-EXAMPLE.js [hunk 4]
```diff
@@ -24,6 +24,21 @@ module.exports = {
     { value: 'WIP', name: 'WIP:      Work in progress' },
   ],
 
+  additionalQuestions: [
```
Comment (leonardoanalista) [inline cz-config-EXAMPLE.js:27]: Thanks for the PR!

I believe the "standard" questions are suitable for most users.

Most people are going to copy and paste this example file. My only concern is they would be overwhelmed by the additional questions. 

The README file explains how to use the new field `additionalQuestions` if they need it. 

So it would LGTM if we leave this file `cz-config-EXAMPLE.js` untouched like a bare minimum to get started. Does it make sense?

Comment (parveen14) [reply cz-config-EXAMPLE.js]: @leonardoanalista , Got it, readme is sufficient enough to explain `additionalQuestions` options.
Removed `additionalQuestions` from this file now

**Comment by parveen14** [comment]
@samarpan-b @leonardoanalista  can we merge  this PR ?

**Review by leonardoanalista — APPROVED** [review]

File: cz-config-EXAMPLE.js (hunk 4 above)
Comment (leonardoanalista) [reply cz-config-EXAMPLE.js]: Thank you @parveen14 and @samarpan-b!
